sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelVersioningTestTest.java

```diff
@@ -1193,6 +1197,44 @@ public void testRequirementR5003NoTestData() {
         assertThrows(NoTestData.class, testClass::testRequirementR5003);
     }
 
+    /**
+     * In the mdib representation on the consumer side in sdc-ri, when processing description modifications, the values
+     * for the MdDescriptionVersion and MdStateVersion are set to the implied value if the description modification
+     * does not contain any values for them. This is the case if the versions are missing in the GetMdibResponse,
+     * or generally for DescriptionModificationReports. If the MdDescriptionVersion or MdStateVersion in the
+     * GetMdibResponse is set to a value other than the implied value, they would be set to the implied value again in the next
+     * DescriptionModificationReport, which would cause the test to fail. As the actual values are not communicated via
+     * reports, the check of the MdDescriptionVersion and MdStateVersion has been removed from the test implementation.
+     * <p>
+     * This unit test ensures that a decrement of the MdDescriptionVersion or MdStateVersion does not cause the test to
+     * fail.
+     * </p>
+     *
+     * @param mdDescriptionVersion the MdDescriptionVersion to set
+     * @param mdStateVersion the MdStateVersion to set
+     *
+     * @throws Exception on any exception
+     */
+    @ParameterizedTest
+    @MethodSource("provideMdDescriptionVersionAndMdStateVersion")
+    public void testRequirementR5003TestMdDescriptionVersionAndMdStateVersion(
+            final @Nullable BigInteger mdDescriptionVersion, final @Nullable BigInteger mdStateVersion)
+            throws Exception {
+        final var initial = buildMdibWithMdDescriptionAndStateVersion(
+                MdibBuilder.DEFAULT_SEQUENCE_ID, mdDescriptionVersion, mdStateVersion);
+        final var firstUpdate = buildDescriptionModificationReportWithParts(
+                SEQUENCE_ID,
```

Comment:
You are using two different SEQUENCE_ID constants, `MdibBuilder.DEFAULT_SEQUENCE_ID` in the previous assignment and `SEQUENCE_ID` here - consolidate

Reply:
fixed